core/publisher.go

```diff
@@ -1,33 +1,33 @@
-package node
+package core
```

Comment:
Love that this is leaving `node`. Wish it were going someplace more specific than `core`. I think we have more than enough functionality to justify a `message` or `messaging` package.

Reply:
@acruikshank At this point `core` is exclusively used for messaging stuff and should be renamed to `message` or `messaging`: #2992